lib/loggers/index.js

```diff
@@ -60,21 +60,30 @@ container.add = function(id) {
   if(!existing) {
     const wrapper = Object.create(logger);
     wrapper.log = function(level, msg, ...meta) {
+      // get default meta to ensure it is always applied
+      const {defaultMeta} = this;
+      // use `Object.getPrototypeOf` because the prototype may change
+      // while the logging subsystem initializes
+      const log = Object.getPrototypeOf(wrapper).log;
       if(arguments.length === 1) {
-        return Object.getPrototypeOf(wrapper).log.apply(
-          wrapper, [level]);
+        return log.apply(wrapper, [{...defaultMeta, ...level}]);
       }
-      return Object.getPrototypeOf(wrapper).log.apply(
-        wrapper, [level, msg, ...meta]);
+      if(meta.length === 0) {
+        meta.push({defaultMeta});
+      } else {
+        meta[0] = {...defaultMeta, ...meta[0]};
+      }
+      return log.apply(wrapper, [level, msg, ...meta]);
     };
-    wrapper.child = function(childMeta) {
+    const createChild = wrapper.child;
+    wrapper.child = function(childMeta, ...args) {
       // simple string name is shortcut for {module: name}
       if(typeof childMeta === 'string') {
         childMeta = {module: childMeta};
       }
-      // create child logger with merged meta data from parent
-      const child = Object.create(this);
-      child.defaultMeta = {...this.defaultMeta, ...childMeta};
+      // create child logger from wrapper and merge child meta into parent meta
+      const child = createChild.apply(wrapper, [childMeta, ...args]);
```

Comment:
We will need to discuss what's going on with the winston `.child` implementation.  I threw out this approach in favor of simply doing the defaultMeta.  The customer `write` implementation in `.child` just gets in the way.